platform/darwin/src/MGLAttributedExpression.h

```diff
@@ -19,9 +32,21 @@ MGL_EXPORT
  */
 @property (strong, nonatomic) NSExpression *expression;
 
+#if TARGET_OS_IPHONE
+/**
+ The formatting attributes dictionary.
+ `MGLFontNamesAttribute` : `NSArray<NSString *>*`
+ `MGLFontSizeAttribute` : `NSNumber`
+ `MGLFontColorAttribute` : `UIColor`
+ */
+#else
 /**
- The formatting attributes.
+ The formatting attributes dictionary.
+ `MGLFontNamesAttribute` : `NSArray<NSString *>*`
+ `MGLFontSizeAttribute` : `NSNumber`
+ `MGLFontColorAttribute` : `NSColor`
  */
+#endif
```

Comment:
I think you may need to duplicate the entire declaration, not just the documentation comment, in order for Quick Help to find the comment.